UIREQ-785: Fix "Timeout of 2000ms exceeded. For async tests and hooks…

pavelspichonak opened the pull request:
## Purpose
Fix "Timeout of 2000ms exceeded. For async tests and hooks, ensure “done()” is called" big tests errors

## Refs
https://issues.folio.org/browse/UIREQ-785

zburke reviewed (changes requested):
The functional change looks fine but please fix the lint error in `karma.conf.js` as lint errors will propagate to all future PRs until they are fixed. The easiest thing is probably to go back to an arrow function. 

Lint wouldn't run for me locally without the following patch, though it seems happy in CI. Weird. Does it run locally for you? 
```
diff --git a/package.json b/package.json
index dbc4b64..a46f6d9 100644
--- a/package.json
+++ b/package.json
@@ -155,7 +155,17 @@
     "@babel/eslint-parser": "^7.17.0",
     "@babel/plugin-proposal-class-properties": "^7.13.0",
     "@babel/plugin-proposal-decorators": "^7.13.5",
+    "@babel/plugin-proposal-export-namespace-from": "^7.17.12",
+    "@babel/plugin-proposal-function-sent": "^7.18.2",
+    "@babel/plugin-proposal-numeric-separator": "^7.16.7",
+    "@babel/plugin-proposal-private-methods": "^7.17.12",
+    "@babel/plugin-proposal-private-property-in-object": "^7.17.12",
+    "@babel/plugin-proposal-throw-expressions": "^7.16.7",
     "@babel/plugin-transform-runtime": "^7.13.10",
+    "@babel/preset-env": "^7.18.2",
+    "@babel/preset-flow": "^7.17.12",
+    "@babel/preset-react": "^7.17.12",
+    "@babel/preset-typescript": "^7.17.12",
     "@bigtest/interactor": "^0.9.1",
     "@bigtest/mocha": "^0.5.2",
     "@bigtest/react": "^0.1.2",
```

pavelspichonak commented:
> The functional change looks fine but please fix the lint error in `karma.conf.js` as lint errors will propagate to all future PRs until they are fixed. The easiest thing is probably to go back to an arrow function.
> 
> Lint wouldn't run for me locally without the following patch, though it seems happy in CI. Weird. Does it run locally for you?
> 
> ```
> diff --git a/package.json b/package.json
> index dbc4b64..a46f6d9 100644
> --- a/package.json
> +++ b/package.json
> @@ -155,7 +155,17 @@
>      "@babel/eslint-parser": "^7.17.0",
>      "@babel/plugin-proposal-class-properties": "^7.13.0",
>      "@babel/plugin-proposal-decorators": "^7.13.5",
> +    "@babel/plugin-proposal-export-namespace-from": "^7.17.12",
> +    "@babel/plugin-proposal-function-sent": "^7.18.2",
> +    "@babel/plugin-proposal-numeric-separator": "^7.16.7",
> +    "@babel/plugin-proposal-private-methods": "^7.17.12",
> +    "@babel/plugin-proposal-private-property-in-object": "^7.17.12",
> +    "@babel/plugin-proposal-throw-expressions": "^7.16.7",
>      "@babel/plugin-transform-runtime": "^7.13.10",
> +    "@babel/preset-env": "^7.18.2",
> +    "@babel/preset-flow": "^7.17.12",
> +    "@babel/preset-react": "^7.17.12",
> +    "@babel/preset-typescript": "^7.17.12",
>      "@bigtest/interactor": "^0.9.1",
>      "@bigtest/mocha": "^0.5.2",
>      "@bigtest/react": "^0.1.2",
> ```

Thanks. Yeah for me linter works locally. Fixed linter warnings. Also I've moved default timeout to constant because it will be used for another pull request in which I fix another type of big test errors related to interactors timeout. And also I've increased timeout to 10 seconds because sometimes one test failed with 6 seconds timeout.
